### PR TITLE
commit for issue #6 -To Edit the blog post by creating edit link #6

### PR DESCRIPTION
**Pull request by Sudhathankavel**
-created edit link using flash method in django
-Edit link has been verified against the secret key.

### CODE REVIEW

File: Blog/templates/Blog/content.html [hunk 1]
```diff
@@ -6,4 +6,14 @@ <h4>Title:</h4>
     <h4>Content:</h4>
     {{blog.content}} <br/><br/>
     <a href="{% url 'Blog:homepage' %}">Back To HomePage</a>
+    {%if messages %}
+        <div class="messages">
+            {% for message in messages %}
+                <div class="" role="alert">
+                 <center>{{message}}</center></a>
```
Comment (aitchnyu) [inline Blog/templates/Blog/content.html]: Its a deprecated tag. And indent it

Comment (Sudhathankavel) [reply Blog/templates/Blog/content.html]: ok will remove that

File: Blog/templates/Blog/content.html [hunk 2]
```diff
@@ -5,5 +5,15 @@ <h4>Title:</h4>
     {{blog.title}}
     <h4>Content:</h4>
     {{blog.content}} <br/><br/>
-    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a>
+    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a><br/><br/>
```
Comment (aitchnyu) [inline Blog/templates/Blog/content.html]: They should be at top of page

File: Blog/templates/Blog/edit_post.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% extends 'Blog/base.html' %}
```
Comment (aitchnyu) [inline Blog/templates/Blog/edit_post.html]: This file is very similar to home page. Avoid duplication and reuse the template. If you want to change message on button, pass the message from view function. If you want home page to have extra features, include this template within home page.

File: Blog/templates/Blog/content.html [hunk 4]
```diff
@@ -1,9 +1,18 @@
 {% extends 'Blog/base.html' %}
 {% block title %} Actual content {% endblock %}
 {% block pagecontent %}
+    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a><br/><br/>
     <h4>Title:</h4>
     {{blog.title}}
     <h4>Content:</h4>
     {{blog.content}} <br/><br/>
-    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a>
+    {%if messages %}
```
Comment (aitchnyu) [inline Blog/templates/Blog/content.html]: Please put messages before the blog content. Users will miss it if they are forced to scroll a long post till the end

**Pull request merged**